pkg/kubelet/kubelet.go

```diff
@@ -1359,6 +1420,10 @@ func (kl *Kubelet) initializeModules() error {
 
 // initializeRuntimeDependentModules will initialize internal modules that require the container runtime to be up.
 func (kl *Kubelet) initializeRuntimeDependentModules() {
+	if err := kl.chownDirForRemappedIDs(kl.getPodsDir()); err != nil {
+		klog.Fatalf("Kubelet failed to change kubelet pod dir ownership to remapped user: %v", err)
+	}
```

Comment:
Is this really needed? I did some tests with these lines commented out and it seems to work fine...

Reply:
I thought you had a problem with:

> Bad owner of /var/lib/kubelet/pods

I think this code aims to fix that, but with a bug. See [comment](https://github.com/kinvolk/kubernetes/pull/3/files/15bf6a07426431088ba4b28c338ead227ab13fb3#r442745570).